js/background.js

```diff
@@ -15,84 +15,86 @@
  */
 
 
-function Background()
-{
-    $this = this;
+function Background() {
+  $this = this;
 
-    // clearing last search on borwser startup
-    localStorage['last_search'] = '';
+  // clearing last search on borwser startup
+  localStorage['last_search'] = '';
 
-    chrome.extension.onMessage.addListener(function(request, sender, callback){
-        console.log(request);
-        if(request.query)
-            return $this.query(request.query, callback);
-        if (request.options) {
-            callback(localStorage);
-        }
-
-        if (request.selection) {
-        
-        }
+  chrome.extension.onMessage.addListener(function(request, sender, callback) {
+    if (request.query)
+      return $this.query(request.query, callback);
+    if (request.options) {
+      callback(localStorage);
+    }
 
-        if (request.current_url) {
-            chrome.tabs.getSelected(function(tab) {
-                console.log(tab);
-                var url = tab.url;
-                callback(url);
-            });
-        }
+    if (request.current_url) {
+      chrome.tabs.getSelected(function(tab) {
+        console.log(tab);
+        var url = tab.url;
+        callback(url);
+      });
+    }
 
-        return true;
-    });
+    return true;
+  });
 
-//  this.menuID = chrome.contextMenus.create({
-//       "title" : "Ask the duck",
-//       "type" : "normal",
-//       "contexts" : ["selection"],
-//       "onclick" : function() {
-//          console.log('clicked!!!'); 
-//       }
-//  });
+  //  this.menuID = chrome.contextMenus.create({
```

Comment:
@Sayanc93 if we are going to update this file, we can just ditch this, it is implemented below.